graphene_django/fields.py

```diff
@@ -154,18 +160,61 @@ def resolve_connection(cls, connection, args, iterable, max_limit=None):
         ):
             args["first"] = max_limit
 
+        if graphene_settings.USE_DATALOADERS:
+            if not hasattr(info.context, "dataloaders"):
+                info.context.dataloaders = {}
+
+            dataloader_key: str = str(info.field_nodes)
+            if dataloader_key not in info.context.dataloaders:
+
+                def load_many(keys):
+                    # keys is a list of tuples of (queryset, start, stop)
+                    qs = keys[0][0].model.objects.none()
+
+                    objects = list(
```

Comment:
Why do you need to evaluate the QuerySet here (with the `list` explicit call) instead of letting the `for object_ in objects:` loop do it for you?

If there's a good reason (perhaps for indexing?), let's add a comment to explain the logic at least.

Reply:
Same, I thought it improved perf but actually it makes no diff, will remove thanks